src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -49,14 +52,15 @@ public static bool IsLambda(SyntaxNode node)
         /// </summary>
         public static SyntaxNode GetLambda(SyntaxNode lambdaBody)
         {
+            Debug.Assert(lambdaBody.Parent is object);
             var lambda = lambdaBody.Parent;
-            if (lambda.Kind() == SyntaxKind.ArrowExpressionClause)
+            if (lambda.IsKind(SyntaxKind.ArrowExpressionClause))
```

Comment:
>IsKind( [](start = 23, length = 7)

Just curious - why change these?

Reply:
The `lambda.Kind()` on line 63 would warn. Then I think I just aligned the check on line 57 for stylistic consistency.

---
In reply to: [441899787](https://github.com/dotnet/roslyn/pull/44266#discussion_r441899787) [](ancestors = 441899787)